src/Node.fs

```diff
@@ -4824,37 +4824,37 @@ module http2 =
 
     type [<AllowNullLiteral>] Http2Server =
         inherit net.Server
-        abstract addListener: ``event``: string * listener: (ResizeArray<obj> -> unit) -> Http2Server
+        abstract addListener: ``event``: string * listener: (ResizeArray<obj option> -> unit) -> Http2Server
```

Comment:
If the object may be null | undefined when passed in to the listener, this is a good change.